Lib/lib2to3/tests/test_parser.py

```diff
@@ -305,6 +303,17 @@ def test_8(self):
                         *g:6, h:7, i=8, j:9=10, **k:11) -> 12: pass"""
         self.validate(s)
 
+    def test_9(self):
```

Comment:
Would be nice to add tests for signatures like:
```
(a,)
(*args,)
(a=1,)
```
and few combinations if they have a separate path in the grammar.

Reply:
LOL, that's one subtle way to signal "your patch doesn't handle the vararg case" ;-) Alright, I added `*args` support, too, and added the relevant tests.